app/views/admin/users/index.html.haml

```diff
@@ -23,10 +23,11 @@
             %td
               = user.id
             %td
-              - if user.confirmed?
-                confirmed
-              - else
-                unconfirmed
+              - if can? :update, user
+                = check_box_tag user.id, user.id, user.confirmed?, method: :patch, |
```

Comment:
You don't need | at the end of the lines.
Put `method: :patch` on its own line too.